pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -120,7 +120,7 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 		} else {
 			break
 		}
-		done, err := kc.IsInstanceDone(instance, nil)
+		done, err := kc.IsInstanceDone(instance, instance.Spec.PlanExecution.PlanName)
```

Comment:
why are you passing in instance as well as field on that instance? 🤔 should we just pass in an instance?

Reply:
Good eye ;) basically, because I don't know how this method will be used: 
1. the passed instance has to be the one returned by `Create/Update/Patch` method because it has the `PlanExecution.PlanName` field set to the right plan
2. if you simply fetch the instance later, the plan might be already done (which is not a big deal), or there might be already _another plan_ (from a later update) in progress

Passing a plan explicitly was an attempt to make the usage of this method slightly safer.